airbyte-integrations/connectors/source-twilio/source_twilio/schemas/available_phone_numbers_local.json

```
@@ -0,0 +1,82 @@
+{
+  "properties": {
+    "friendly_name": {
+      "type": ["null", "string"]
+    },
+    "phone_number": {
+      "type": ["null", "string"]
+    },
+    "lata": {
+      "type": ["null", "string"]
+    },
+    "rate_center": {
+      "type": ["null", "string"]
+    },
+    "latitude": {
+      "anyOf": [
```

```suggestion
      "type": ["null", "number", "string"]
```
For something we need to complicate the scheme using `anyOf` in this case?